flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_ATC_ConnectionNotification.js

```diff
@@ -1,30 +1,72 @@
 class CDUAtcConnectionNotification {
-    static ShowPage(mcdu, store = {"atcCenter": ""}) {
+    static ShowPage(mcdu, store = {"atcCenter": "", "logonAllowed": false, "loginState": 0, "notifTime": ""}) {
         mcdu.clearDisplay();
-        const canNotify = "";
+        mcdu.page.Current = mcdu.page.ATCNotification;
```

Comment:
because I check on LSK2 and RSK2 in asynchronous functions if the page is still active.
if I don't do this and leave the page, the code returns after the asynchronous call is finished